socialite/server/graphql/resolvers/edit.js

```diff
@@ -0,0 +1,18 @@
+const {User, Profile, UserDets, UserSub} = require("../../models/User");
+const data = require("../../util/userdata");
+const {scoring, common, resetratio} = require('../../recosys/content')
+
+
+module.exports = {
+	edit: {
+		async edit(_, { user_id, hou, hsnum, hsname, sprts, clbs }) {
```

Comment:
Instead of taking each field separately, maybe assume front-end passes all in a single json object. For this make a separate object type in typedefs and take that type as input. Makes code cleaner, that's all.

Reply:
pls review again